site/app/Resources/views/notification/notification_inbox.html.twig

```diff
@@ -0,0 +1,77 @@
+{% extends 'layout.html.twig' %}
+
+{% block title %}
+    Notification Inbox
+{% endblock %}
+
+{% block pagepath %}Home &raquo; Notification Inbox{% endblock %}
+
+{% block stylesheets %}
+    {{ parent() }}
+    {{ encore_entry_link_tags('notification_inbox') }}
+{% endblock %}
+
+{% block content %}
+    <h1>Notification Inbox</h1>
+    <div class="inbox-header row">
+        <div class="col-xs-6 col-md-6 col-sm-6 inbox-title">
+            Your Notifications ({{ mgilet_notification_count(app.user) }})
+        </div>
+        <div class="col-xs-6 col-md-6 col-sm-6 text-right">
+            <a class="inbox-settings" href="/notification/settings">Notification Settings</a>
+        </div>
+    </div>
+    <ul class="all-notifications">
+        {# If there are any notifications #}
+        {% if (mgilet_notification_count(app.user)) %}
+            {% for notification in notificationList %}
+                {% if (notification.seen) == true %}
+                    {# Render Seen Notification #}
+                    <li class="inbox-seen">
+                        <div class="row">
+                            <div class="col-sm-8 col-lg-8 col-xs-8 col-sm-8 inbox-message">
+                                {{ notification.notification.message }}
+                            </div>
+                            <button class="col-xs-4 col-md-4 col-sm-4 text-right notification-remove"
```

Comment:
That column is way too large with 4/12 of the screen. You need a fixed-size column for the remove icon.